### PR TITLE
ARTESCA-11662: trigger veeam assistant from account list page

### PR DESCRIPTION
**Pull request by ChengYanJin**
Light theme: 
image
image

Dark theme:
image
image

### CODE REVIEW

File: src/react/ui-elements/Veeam/VeeamConfiguration.test.tsx [hunk 1]
```diff
@@ -31,25 +39,52 @@ describe('Veeam Configuration UI', () => {
...
   };
 
-  it('should be able to set the Veeam configuration', async () => {
-    //S
-    render(
-      <QueryClientProvider client={new QueryClient()}>
-        <Stepper
-          steps={[
+  const mockUseAccountsImplementation = () => {
+    //@ts-ignore
```
Comment (hervedombya) [inline src/react/ui-elements/Veeam/VeeamConfiguration.test.tsx]: cast `mockUseAccountsLocationsAndEndpoints` as jest.Mock to not have the ts-error anymore

File: src/react/ui-elements/Veeam/VeeamConfiguration.tsx [hunk 2]
```diff
@@ -125,12 +131,35 @@ const Configuration = () => {
...
   const { next } = useStepper(VeeamStepsIndexes.Configuration, VEEAM_STEPS);
   const application = watch('application');
+  const accountName = watch('accountName');
+
+  const isAccountExist = useMemo(() => {
+    return (
+      status === 'success' &&
+      accounts.some((account) => account.name === accountName)
+    );
+  }, [accountName, status]);
```
Comment (hervedombya) [inline src/react/ui-elements/Veeam/VeeamConfiguration.tsx]: `accounts` are missing in deps

**Review by hervedombya — APPROVED** [review]
LGTM !

File: src/react/ui-elements/Veeam/VeeamConfiguration.tsx [hunk 3]
```diff
@@ -34,40 +34,69 @@ import { VEEAM_STEPS, VeeamStepsIndexes } from './VeeamSteps';
...
 import { getCapacityBytes } from './useCapacityUnit';
 import { bucketNameValidationSchema } from '../../databrowser/buckets/BucketCreate';
+import { accountNameValidationSchema } from '../../../react/account/AccountCreate';
+import { useAccountsLocationsAndEndpoints } from '../../../react/next-architecture/domain/business/accounts';
+import { useAccountsLocationsEndpointsAdapter } from '../../..//react/next-architecture/ui/AccountsLocationsEndpointsAdapterProvider';
+import { VeeamLogo } from './VeeamLogo';
+
+const hasMoreThanOneDecimal = (num: number) => {
+  const decimalCount = num.toString().split('.')[1]?.length || 0;
+  return decimalCount > 1;
```
Comment (JBWatenbergScality) [inline src/react/ui-elements/Veeam/VeeamConfiguration.tsx:44]: ```Math.floor(num*10) === num*10```

**Pull request merged**